databricks/koalas/testing/utils.py

```diff
@@ -272,6 +273,25 @@ def temp_file(self):
         with self.temp_dir() as tmp:
             yield tempfile.mktemp(dir=tmp)
 
+    @contextmanager
+    def cd(self, newdir, cleanup=lambda: True):
```

Comment:
I couldn't get why we need these two.
Didn't `temp_dir` work for any reason?

Reply:
Removing my functions and adding `temp_dir`.
Thanks for pointing out.